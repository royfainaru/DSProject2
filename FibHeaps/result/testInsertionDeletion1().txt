╮
├── ╭────────────────╮
│   │ Key: 4         │
│   ├────────────────┤
│   │ Rank: 3        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: (null)   │
│   │ Child: 5       │
│   ╰────────────────╯
│   ├── ╭──────────────╮
│   │   │ Key: 5       │
│   │   ├──────────────┤
│   │   │ Rank: 1      │
│   │   │ Marked: true │
│   │   │ Parent: 4    │
│   │   │ Next: 6      │
│   │   │ Prev: (null) │
│   │   │ Child: 10    │
│   │   ╰──────────────╯
│   │   ╰── ╭───────────────╮
│   │       │ Key: 10       │
│   │       ├───────────────┤
│   │       │ Rank: 1       │
│   │       │ Marked: false │
│   │       │ Parent: 5     │
│   │       │ Next: (null)  │
│   │       │ Prev: (null)  │
│   │       │ Child: 11     │
│   │       ╰───────────────╯
│   │       ╰── ╭───────────────╮
│   │           │ Key: 11       │
│   │           ├───────────────┤
│   │           │ Rank: 0       │
│   │           │ Marked: false │
│   │           │ Parent: 10    │
│   │           │ Next: (null)  │
│   │           │ Prev: (null)  │
│   │           │ Child: (null) │
│   │           ╰───────────────╯
│   ├── ╭───────────────╮
│   │   │ Key: 6        │
│   │   ├───────────────┤
│   │   │ Rank: 1       │
│   │   │ Marked: false │
│   │   │ Parent: 4     │
│   │   │ Next: 7       │
│   │   │ Prev: 5       │
│   │   │ Child: 8      │
│   │   ╰───────────────╯
│   │   ╰── ╭───────────────╮
│   │       │ Key: 8        │
│   │       ├───────────────┤
│   │       │ Rank: 0       │
│   │       │ Marked: false │
│   │       │ Parent: 6     │
│   │       │ Next: (null)  │
│   │       │ Prev: (null)  │
│   │       │ Child: (null) │
│   │       ╰───────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 7        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 4     │
│       │ Next: (null)  │
│       │ Prev: 6       │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 1        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 4        │
    │ Child: 3       │
    ╰────────────────╯
    ╰── ╭───────────────╮
        │ Key: 3        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 2     │
        │ Next: (null)  │
        │ Prev: (null)  │
        │ Child: (null) │
        ╰───────────────╯
