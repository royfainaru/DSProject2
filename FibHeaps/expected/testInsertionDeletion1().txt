╮
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 1        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 4        │
│   │ Prev: 4        │
│   │ Child: 3       │
│   ╰────────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 3        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 2     │
│       │ Next: 3       │
│       │ Prev: 3       │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 4         │
    ├────────────────┤
    │ Rank: 3        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 2        │
    │ Prev: 2        │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭──────────────╮
    │   │ Key: 5       │
    │   ├──────────────┤
    │   │ Rank: 1      │
    │   │ Marked: true │
    │   │ Parent: 4    │
    │   │ Next: 6      │
    │   │ Prev: 7      │
    │   │ Child: 10    │
    │   ╰──────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 10       │
    │       ├───────────────┤
    │       │ Rank: 1       │
    │       │ Marked: false │
    │       │ Parent: 5     │
    │       │ Next: 10      │
    │       │ Prev: 10      │
    │       │ Child: 11     │
    │       ╰───────────────╯
    │       ╰── ╭───────────────╮
    │           │ Key: 11       │
    │           ├───────────────┤
    │           │ Rank: 0       │
    │           │ Marked: false │
    │           │ Parent: 10    │
    │           │ Next: 11      │
    │           │ Prev: 11      │
    │           │ Child: (null) │
    │           ╰───────────────╯
    ├── ╭───────────────╮
    │   │ Key: 6        │
    │   ├───────────────┤
    │   │ Rank: 1       │
    │   │ Marked: false │
    │   │ Parent: 4     │
    │   │ Next: 7       │
    │   │ Prev: 5       │
    │   │ Child: 8      │
    │   ╰───────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 8        │
    │       ├───────────────┤
    │       │ Rank: 0       │
    │       │ Marked: false │
    │       │ Parent: 6     │
    │       │ Next: 8       │
    │       │ Prev: 8       │
    │       │ Child: (null) │
    │       ╰───────────────╯
    ╰── ╭───────────────╮
        │ Key: 7        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 4     │
        │ Next: 5       │
        │ Prev: 6       │
        │ Child: (null) │
        ╰───────────────╯
